01_basic_stream/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+<script type="module">
+	import {AnboxStream, AnboxStreamGatewayConnector} from './anbox-stream-sdk.js';
+
+	const connector = new AnboxStreamGatewayConnector({
+		url: 'https://gateway-url.net',
+		authToken: 'YOUR_AUTH_TOKEN',
```

Comment:
Can we somehow get both URL and auth token passed in via docker env variables? That would make it straight forward and simple.